tools/depth-quality/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ find_package(Threads REQUIRED)
 include_directories(${CMAKE_BINARY_DIR})
 
 include("${REPO_ROOT}/include/librealsense2/utilities/easylogging/easyloggingpp.cmake")
+target_sources( ${PROJECT_NAME} PRIVATE ${ELPP_SOURCE_FILES}  )
```

Comment:
Why was this needed here? We didn't change something at the DQT side right?

Reply:
You're right that it may be removable. Can we push my investigation of this to the next PR?

Reply:
Sure